### ISSUE TITLE
[v22] pending member initiator is not populated

### ISSUE THREAD

**Issue by skjnldsv** [nextcloud/circles#647]
Currently we don't have a way of knowing if the user have already requested joining a circle.
I would assume initiator would be populated and level 0 ?

**Comment by ArtificialOwl** [comment]
Was not originally done for this purpose, but this PR should fix your issue: #637 

Now, initiator contains `level=0` and `status="Requesting"`
```
{
    "id": "YPqqoVpMcNrwoRKo8tMSYgfXsswi4OM",
    "name": "test 0001",
    "displayName": "test 0001",
    "sanitizedName": "test 0001",
    "source": 16,
    "population": 2,
    "config": 88,
    "description": "",
    "settings": [],
    "url": "\/index.php\/apps\/contacts\/direct\/circle\/YPqqoVpMcNrwoRKo8tMSYgfXsswi4OM",
    "creation": 1624894144,
    "initiator": {
        "id": "6BxblqZAJEYGIC1qjWi9xZ2inUMyjty",
        "circleId": "YPqqoVpMcNrwoRKo8tMSYgfXsswi4OM",
        "singleId": "LnpZritg3Wce9TbOKRkdOkYTJE87v28",
        "userId": "test1",
        "userType": 1,
        "instance": "migration.local",
        "local": true,
        "level": 0,
        "status": "Requesting",
        "displayName": "test1",
       [...]
```

**Comment by skjnldsv** [comment]
https://github.com/nextcloud/contacts/pull/2316
